bokehjs/src/coffee/embed.ts

```diff
@@ -66,15 +77,21 @@ function _handle_notebook_comms(this: Document, receiver: Receiver, comm_msg: Co
 function _update_comms_callback(target: string, doc: Document, comm: Comm): void {
   if (target == comm.target_name) {
     const r = new Receiver()
-    comm.on_msg(_handle_notebook_comms.bind(doc, r))
+    if (comm.on_msg) {
+      // Classic Notebook
+      comm.on_msg(_handle_notebook_comms.bind(doc, r))
+    } else {
+      // JupyterLab
+      comm.onMsg = _handle_notebook_comms.bind(doc, r)
+    }
```

Comment:
Amazing, this spelling difference is a back door way to distinguish classic notebook from jlab?

Reply:
Yes, there's a bunch of API differences you could use but this seemed simplest.